agent/consul/options.go

```diff
@@ -1,6 +1,7 @@
 package consul
 
 import (
+	"github.com/hashicorp/consul/lib"
```

Comment:
NIT: needs to be moved down and grouped with the other `github.com/hashicorp/consul` imports.

Reply:
thanks for catching! fixed